Packs/GitHub/Integrations/GitHub/GitHub.py

```diff
@@ -1309,6 +1323,25 @@ def get_workflow_usage(owner_name, repository_name, workflow_id):
     return workflow_usage
 
 
+def get_team_members_command():
+    args = demisto.args()
+    org = args.get('organization')
+    team_slug = args.get('team_slug')
+    response = get_team_members(org, team_slug)
+    members = []
+    for member in response:
+        ec_object = {
+            'ID': member.get("id"),
+            'Login': member.get("login"),
+        }
+        members.append(ec_object)
+    ec = {
+        'GitHub.TEAMMEMBER(val.ID === obj.ID)': members
```

Comment:
I think we should have some kind of identifier to which team the member belongs to.
In the way it is presented now - it's just going to be a list of users.